benchmarks/step_detect.py

```diff
@@ -17,4 +17,4 @@ def time_detect_regressions(self):
             step_detect.detect_regressions(self.y)
 
     def time_solve_potts_approx(self):
-        step_detect.solve_potts_approx(self.y, 0.3, p=1)
+        step_detect.solve_potts_approx(self.y, [0.3] * len(self.y), gamma=1)
```

Comment:
I don't understand how that change relates to the PR.

Reply:
This is actually https://github.com/airspeed-velocity/asv/pull/779, which came about from my wanting to use `asv`'s own suite to demonstrate this change - but the one benchmark in the vicinity of `15.6ms` was broken due to a typo. It's necessary to reproduce the results above, but I will remove prior to merge.